script.js

```diff
@@ -22,16 +31,161 @@ ulCont.addEventListener('click', ((e) => {
+  projectLanguagesRuby[i].innerHTML = storeData[i].technologies[1];
+  projectLanguagesCss[i].innerHTML = storeData[i].technologies[2];
+  projectLanguagesJs[i].innerHTML = storeData[i].technologies[3];
+  projectsDescription[i].innerHTML = storeData[i].description;
+}
+
+const popupsProjectImage = document.querySelector('.popup-img');
+const popupsProjectTitle = document.querySelector('.popups-proj-title');
+const popupsProjectDesktop = document.querySelector('.popups-project-p-desktop');
+const popupsProjectMobile = document.querySelector('.popups-project-p-mobile');
+const tagsUp = document.querySelectorAll('.card-tagsLi');
+const tagsDown = document.querySelectorAll('.card-tagsLi1');
+
+const openModal = function () {
+  overlay.classList.remove('hidden');
+  modal.style.display = 'flex';
+  // eslint-disable-next-line no-use-before-define
+  popupsProjectImage.src = storeData[0].imagesrc2;
+  popupsProjectTitle.innerHTML = storeData[0].name;
+  popupsProjectDesktop.innerHTML = storeData[0].content1;
+  popupsProjectMobile.innerHTML = storeData[0].content2;
+  tagsUp.innerHTML = storeData[0].tags1;
+  tagsDown.innerHTML = storeData[0].tags2;
+  document.documentElement.scrollTop = 0;
+};
+
+openModalBtn.addEventListener('click', openModal);
+
+const openModal2 = function () {
+  overlay.classList.remove('hidden');
+  modal.style.display = 'flex';
+  // eslint-disable-next-line no-use-before-define
+  popupsProjectImage.src = storeData[1].imagesrc;
+  popupsProjectTitle.innerHTML = storeData[1].name;
+  popupsProjectDesktop.innerHTML = storeData[1].content1;
+  popupsProjectMobile.innerHTML = storeData[1].content2;
+  popupsProjectTitle.style.fontSize = '20px';
+  document.documentElement.scrollTop = 0;
+};
+
+openModalBtn2.addEventListener('click', openModal2);
+
+const openModal3 = function () {
+  overlay.classList.remove('hidden');
+  modal.style.display = 'flex';
+  // eslint-disable-next-line no-use-before-define
+  popupsProjectImage.src = storeData[2].imagesrc;
+  popupsProjectTitle.innerHTML = storeData[2].name;
+  popupsProjectDesktop.innerHTML = storeData[2].content1;
+  popupsProjectMobile.innerHTML = storeData[2].content2;
+  popupsProjectTitle.style.fontSize = '20px';
+  document.documentElement.scrollTop = 0;
+};
+
+openModalBtn3.addEventListener('click', openModal3);
+
+const openModal4 = function () {
+  overlay.classList.remove('hidden');
+  modal.style.display = 'flex';
+  // eslint-disable-next-line no-use-before-define
+  popupsProjectImage.src = storeData[3].imagesrc;
+  popupsProjectTitle.innerHTML = storeData[3].name;
+  popupsProjectDesktop.innerHTML = storeData[3].content1;
+  popupsProjectMobile.innerHTML = storeData[3].content2;
+  document.documentElement.scrollTop = 0;
+};
+
+openModalBtn4.addEventListener('click', openModal4);
+
+const closeModal = function () {
+  modal.style.display = 'none';
+  overlay.classList.add('hidden');
+};
+
+closeModalBtn.addEventListener('click', closeModal);
+
+// Local Storage
```

Comment:
- [ ] Kindly add a function when the user loads the page, the code checks if there's any saved data in the local storage and if so the input fields are pre-filled with the data. Check the example below.
![image](https://user-images.githubusercontent.com/81932478/210356005-8a61f406-ee7e-4df4-8d86-582e71f50884.png)